src/igvfd/audit/matrix_file.py

```diff
@@ -11,11 +11,11 @@
 @audit_checker('MatrixFile', frame='object')
 def audit_matrix_file_dimensions(value, system):
     '''
-        audit_detail: Matrix files are expected to have different values for each dimension.
+        audit_detail: Non-hic matrix files are expected to have different values for each dimension.
```

Comment:
please change to "Matrix files, with the exception of .hic files are expected..."

Reply:
done